build/buildBuildImages.sh

```diff
@@ -397,6 +397,45 @@ function buildCliImage() {
 	echo "$builtImageName" >> $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 }
 
+function buildAndRunImage() {
```

Comment:
Should this be renamed to `buildFullImage`?

Reply:
This script already has a `full` tag parameter. So I'll have to rename that parameter or figure out a new name for this. I shouldn't add this with the full parameter since it builds all docker images (jamstack, lts-versions, etc...)

Reply:
Thanks, so what is the plan there moving forward? Do you want to keep the name "buildAndRun"? Do you want to move to "full" later after we rename?
Also, even if we keep this, should we at least rename to `buildBuildAndRunImage`?

Reply:
I've updated the image name for this PR.